{"uid":"53a2059d848bf0c0","name":"test_log_rotation","fullName":"tests.integration.test_log_rotation#test_log_rotation","historyId":"cee6faeba82de98eaf891ec9dd9520e1","time":{"start":1722737919530,"stop":1722738821593,"duration":902063},"description":"Test log rotation.","descriptionHtml":"<p>Test log rotation.</p>\n","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 130, in _block\n    await jasyncio.sleep(wait_period)\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 605, in sleep\n    return await future\nasyncio.exceptions.CancelledError\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 456, in wait_for\n    return fut.result()\nasyncio.exceptions.CancelledError\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 177, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 169, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1788, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 194, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/tests/integration/test_log_rotation.py\", line 81, in test_log_rotation\n    await ops_test.model.block_until(\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1040, in block_until\n    await utils.block_until(done,\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 131, in block_until\n    await jasyncio.shield(jasyncio.wait_for(_block(), timeout))\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 458, in wait_for\n    raise exceptions.TimeoutError() from exc\nasyncio.exceptions.TimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1722737919248,"stop":1722737919248,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1722737919248,"stop":1722737919248,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1722737919248,"stop":1722737919529,"duration":281},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1722737919529,"stop":1722737919530,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Test log rotation.","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 130, in _block\n    await jasyncio.sleep(wait_period)\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 605, in sleep\n    return await future\nasyncio.exceptions.CancelledError\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 456, in wait_for\n    return fut.result()\nasyncio.exceptions.CancelledError\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 177, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 169, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1788, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 152, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 114, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 194, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/tests/integration/test_log_rotation.py\", line 81, in test_log_rotation\n    await ops_test.model.block_until(\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1040, in block_until\n    await utils.block_until(done,\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 131, in block_until\n    await jasyncio.shield(jasyncio.wait_for(_block(), timeout))\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 458, in wait_for\n    raise exceptions.TimeoutError() from exc\nasyncio.exceptions.TimeoutError","steps":[],"attachments":[{"uid":"a3e4e6c5092b6ef2","name":"log","source":"a3e4e6c5092b6ef2.txt","type":"text/plain","size":2134}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1722738821597,"stop":1722738821597,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1722738821597,"stop":1722738829611,"duration":8014},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1722738829611,"stop":1722738829611,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1722738829611,"stop":1722738829611,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1722738829611,"stop":1722738829612,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_log_rotation"},{"name":"host","value":"fv-az1984-724"},{"name":"thread","value":"12178-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_log_rotation"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"48701ec245a520d0","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":3,"skipped":0,"passed":63,"unknown":1,"total":67},"items":[{"uid":"c6568bd00da3746b","reportUrl":"../190//#testresult/c6568bd00da3746b","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1722651117666,"stop":1722652020716,"duration":903050}},{"uid":"be8277fa5be16b4","reportUrl":"../188//#testresult/be8277fa5be16b4","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1722564731882,"stop":1722565634214,"duration":902332}},{"uid":"6e8b2dc80cbb447b","reportUrl":"../186//#testresult/6e8b2dc80cbb447b","status":"passed","time":{"start":1722478665534,"stop":1722478774171,"duration":108637}},{"uid":"baa97c7a16587d49","reportUrl":"../184//#testresult/baa97c7a16587d49","status":"passed","time":{"start":1722390558805,"stop":1722390674334,"duration":115529}},{"uid":"ee95a73c20f9e0ff","reportUrl":"../181//#testresult/ee95a73c20f9e0ff","status":"passed","time":{"start":1722305644850,"stop":1722305764820,"duration":119970}},{"uid":"19bba385ddf64eee","reportUrl":"../177//#testresult/19bba385ddf64eee","status":"passed","time":{"start":1722219191858,"stop":1722219290843,"duration":98985}},{"uid":"71a8e5e586a99434","reportUrl":"../176//#testresult/71a8e5e586a99434","status":"unknown","time":{}},{"uid":"f1ecf6e498d143a8","reportUrl":"../175//#testresult/f1ecf6e498d143a8","status":"passed","time":{"start":1722046135214,"stop":1722046249427,"duration":114213}},{"uid":"fe9d564803b11fb9","reportUrl":"../174//#testresult/fe9d564803b11fb9","status":"passed","time":{"start":1721959839210,"stop":1721959954752,"duration":115542}},{"uid":"f0bdaaeca1740aad","reportUrl":"../172//#testresult/f0bdaaeca1740aad","status":"passed","time":{"start":1721787399163,"stop":1721787501517,"duration":102354}},{"uid":"127a0032aa4021d3","reportUrl":"../169//#testresult/127a0032aa4021d3","status":"passed","time":{"start":1721700664155,"stop":1721700784271,"duration":120116}},{"uid":"239dd1ebae1d0815","reportUrl":"../166//#testresult/239dd1ebae1d0815","status":"passed","time":{"start":1721614461998,"stop":1721614562641,"duration":100643}},{"uid":"2c33ce5989e580bb","reportUrl":"../165//#testresult/2c33ce5989e580bb","status":"passed","time":{"start":1721528078375,"stop":1721528187524,"duration":109149}},{"uid":"ad9cdcdea600ad35","reportUrl":"../164//#testresult/ad9cdcdea600ad35","status":"passed","time":{"start":1721441242456,"stop":1721441341350,"duration":98894}},{"uid":"1506ddb07e0f886d","reportUrl":"../163//#testresult/1506ddb07e0f886d","status":"passed","time":{"start":1721355028387,"stop":1721355138809,"duration":110422}},{"uid":"29e93bce71faf4a9","reportUrl":"../161//#testresult/29e93bce71faf4a9","status":"passed","time":{"start":1721268875757,"stop":1721268982835,"duration":107078}},{"uid":"3e74206a3eb2ea2a","reportUrl":"../158//#testresult/3e74206a3eb2ea2a","status":"passed","time":{"start":1721182499430,"stop":1721182601695,"duration":102265}},{"uid":"6421c6f946f86e49","reportUrl":"../157//#testresult/6421c6f946f86e49","status":"passed","time":{"start":1721095915330,"stop":1721096016852,"duration":101522}},{"uid":"edbf1d0c59c3fbf3","reportUrl":"../156//#testresult/edbf1d0c59c3fbf3","status":"passed","time":{"start":1721009689351,"stop":1721009790138,"duration":100787}},{"uid":"b2c03197e0a3f59e","reportUrl":"../155//#testresult/b2c03197e0a3f59e","status":"passed","time":{"start":1720923454598,"stop":1720923561421,"duration":106823}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"53a2059d848bf0c0.json","parameterValues":[]}